README.md

```diff
@@ -26,16 +26,22 @@ Some systems provide pre-built packages:
 
 * On Debian 9 and Ubuntu 16.04 or newer:
 
-```
-sudo apt-get install s3fs
-```
+  ```
+  sudo apt-get install s3fs
+  ```
+
+* On SUSE 12 or newer and openSUSE 42.1 or newer:
+
+  ```ShellSession
```

Comment:
Could you remove the `ShellSession` to match the other examples?

Reply:
Done.